File: trunk/C#/RBD/RBD/DB/DumpTableException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.IO;

namespace RBD.DB
{
    public class DumpTableException : Exception 
    {
 
        private string errorMessage;
        
        public DumpTableException(IOException e)
        {
                errorMessage = e.getMessage();
        }
        
        public String getErrorMessage()
        {
                return errorMessage;
        }
    }
}

File: SR-RBD/src/ddb/db/DumpTableException.java
package ddb.db;

import java.io.IOException;

public class DumpTableException extends Exception 
{
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	private String errorMessage;
	
	public DumpTableException(IOException e)
	{
		errorMessage = e.getMessage();
	}
	
	public String getErrorMessage()
	{
		return errorMessage;
	}
}
